components/HomepageNews/HomepageNews.vue

```diff
@@ -18,13 +18,15 @@
           <div class="sparc-card__detail">
             <svg-icon name="icon-calendar" height="16" width="16" />
             <p>{{ eventDate(item) }}</p>
-            <svg-icon
-              class="sparc-card__detail--location"
-              name="icon-map"
-              height="16"
-              width="16"
-            />
-            <p>{{ item.fields.location }}</p>
+            <div v-if="item.fields.location">
+              <svg-icon
+                class="sparc-card__detail--location"
+                name="icon-map"
+                height="16"
+                width="16"
+              />
+              <p>{{ item.fields.location }}</p>
+            </div>
```

Comment:
You can use the `template` tag to wrap conditional templates. This result in no additional markup being rendered, and just the markup inside of the `template`. The parent here, `.sparc-card__detail`, is a flexbox item, and this new `div` may break some of the flex alignment.
```suggestion
            <template v-if="item.fields.location">
              <svg-icon
                class="sparc-card__detail--location"
                name="icon-map"
                height="16"
                width="16"
              />
              <p>{{ item.fields.location }}</p>
            </template>
```